src/host/lapack_api.hpp

```diff
@@ -67,10 +70,28 @@ inline auto dlamch(char cmach) -> double {
 #endif
 }
 
+inline auto set_omp_num_threads() -> int {
+  int env_omp_num_threads = 1;
```

Comment:
These OpenMP changes break the non-OpenMP builds and not all LAPACK implementation use OpenMP for multi-threading. I'd strongly suggest to just revert all changes to this file.

Reply:
Reverted in through 6f1ade6 for now. I'll revisit the case separately.